build/StablePackagesInfo.props

```diff
@@ -0,0 +1,14 @@
+<Project>
+    <ItemGroup>
+        <StableProject Include="Microsoft.ML"/>
```

Comment:
Note there are a few assemblies inside of this nuget package - so we will want to put all those assemblies in this list as well. Like `ML.Data`, `ML.Transforms`, `ML.StandardTrainers`.